src/manage_sql.c

```diff
@@ -28602,19 +28602,28 @@ manage_send_report (report_t report, report_t delta_report,
       alert_t alert = 0;
       alert_condition_t condition;
       alert_method_t method;
+      get_data_t get_copy;
 
       if (find_alert_with_permission (alert_id, &alert, "get_alerts"))
         return -1;
 
       if (alert == 0)
         return 1;
 
+      get_copy = *get;
       condition = alert_condition (alert);
       method = alert_method (alert);
+      if (method == ALERT_METHOD_EMAIL)
+        {
+          get_copy.filter = g_strdup_printf ("notes=1 overrides=1"
+                                             " sort-reverse=severity rows=1000");
+	  get_copy.ignore_pagination = 0;
+	}
```

Comment:
I think this should apply to all cases where an alert is used and the filter should be set in the function calling `manage_send_report` if an alert_id is set. In that case a there may also be a filter defined by the alert that should be used.